main.py

```diff
@@ -0,0 +1,23 @@
+from pretrained_models import models
+from featurevis import featurevis, images, image_reader
+import argparse
+
+parser = argparse.ArgumentParser()
+
+parser.add_argument("-a", "--architecture", type = str, default = "inceptionV3", help = "The model architecture")
+args = parser.parse_args()
+
+arch = args.architecture
+#architecture = "inceptionV3"
+
+model = models.get_model(arch)
+# model.summary()
+image = images.initialize_image(224, 224)
+loss, image, layer_name, channel_num = featurevis.visualize_filter(image, model, "mixed6", 1, 500, 2, 0, 0, 0)
+
+name = "feature_vis_{}_{}_{}".format(arch, layer_name, channel_num)
+print(loss)
+
+images.save_image(image, name= name)
+image_reader.save_npy_as_png("{}.npy".format(name))
+
```

Comment:
I think we should not have a `main.py` like this. If people are supposed to use this as a library (as is probably intended), i.e., to write their own code and import this as a package (`import luna`), then such a file is not needed. What we could think of is two options:

1. Currently, we have a how-to demo in the README.md. It could stay this way and we could change that readme if we think this code is a better showcase.
2. We could do a `luna_demo.py`, which does what you have here. The downside of this is that when the project is used as a library, this does not really make sense, as there is no method to call.

These things considered, I would move this code to the README.md, if we think this is a better showcase. What do you think?

Reply:
I would agree more to put it inside README.md. Maybe we can talk more about it in our meeting.

Reply:
Sure, lets talk tomorrow.